feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -68,12 +68,17 @@ func TestCopyingDebugFiles(t *testing.T) {
 	t.Logf("gnoiClient.System().KillProcess() response: %v, err: %v", processKillResponse, err)
 	t.Logf("Wait 60 seconds for process to restart ...")
 	time.Sleep(60 * time.Second)
-
+	
+	chassisMap := map[string]string{"name": "Chassis"}
 	req := &hpb.GetRequest{
 		Path: &tpb.Path{
 			Elem: []*tpb.PathElem{
 				{
-					Name: "components/component/chassis",
+					Name: "components",
+				},
+				{
+					Name: "component",
+					Key:  chassisMap,
```

Comment:
minor: to avoid confusion, either
* rename `chassisMap` to `componentName`
* or simply use `map[string]string{"name": "Chassis"}` here without the variable (it's only used once anyway)

Reply:
yeah I'll just rename to `componentName`

Reply:
done @xw-g